### DISCUSSION TITLE
为什么不支持Redis的审计，是否有难点？

### DISCUSSION THREAD

**Discussion by matrixoom** [hhyo/Archery#2268] [Q&A]
你好，我们在使用您开源的sql审计平台Archery，非常感谢您的贡献。使用过程中有一点不太方便的是：不支持redis的审计和上线，请问有技术难点吗？还是仅仅是没有时间或必要做呢？

**Comment by LeoQuote** [comment]
主要是没人做, 欢迎 pr

**Comment by matrixoom** [comment]
您好，您的来信我已收到，谢谢。
